.github/workflows/ci.yml

```diff
@@ -0,0 +1,176 @@
+on:
+  # Run on pushes to tags, the "master" branch, and PR's
+  push:
+    tags-ignore:
+    branches:
+      - master
+  pull_request:
+
+name: CI
+
+jobs:
+  # Job key
+  ci:
+    # Label displayed in UI
+    name: CI
+    runs-on: ${{ matrix.os }}
+    strategy:
+      # Don't cancel the macOS build if the Linux build fails, etc.
+      fail-fast: false
+      matrix:
+        os: [ubuntu-16.04, macos-10.15, windows-2019]
```

Comment:
Why also build windows? Do you want to move away from appveyor as well?

Reply:
Yeah, sorry, I should have mentioned that.
Originally we were discussing staying on appveyor - once I started doing this port, I realized I want to move everything to Github actions. The biggest reason is to have only one CI provider to have to fix when it breaks, rather than two. The other benefit is x86 and win64 Windows builds get built in parallel now.

Reply:
Ah, I see. Yeah that sounds like it's an improvement!

Reply:
Yeah - the full Github Actions build of 6 different configurations finishes in 16 minutes. Not perfect, but better than before!